test/binary/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+###############################################################################
+# Top contributors (to current version):
+#   Aina Niemetz, Andrew V. Jones, Gereon Kremer
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# The build system configuration.
+##
+include_directories(.)
+include_directories(${PROJECT_SOURCE_DIR}/src)
+include_directories(${PROJECT_SOURCE_DIR}/src/include)
+include_directories(${CMAKE_BINARY_DIR}/src)
+
+add_custom_target(binarytests
```

Comment:
This should be guarded (moved below). If no pexpect, the target will not find any test with label `binary`.

Reply:
Addressed in a8426434ad163d00f0207da706960459bcfa4738 .